platzky/models.py

```
@@ -55,16 +97,47 @@ class Post(BaseModel):
     date: str
 
     def __lt__(self, other):
```

Ruff (0.14.10)

Missing return type annotation for special method `__lt__`

(ANN204)